root_github.tf

```diff
@@ -38,6 +33,16 @@ module "github_e2e_tests_environment" {
   }
 }
 
+module "github_transfer_frontend_environment" {
+  source          = "./tdr-terraform-modules/github_environments"
+  environment     = local.environment
+  repository_name = "nationalarchives/tdr-file-format"
```

Comment:
Is that the correct repo name?

Reply:
Nope :grin: I was testing it on that repo and forgot to change it.